tools/testers/algorithm-tester.pl

```diff
@@ -144,18 +144,8 @@ sub Usage {
         $psql = "\"$psql\"";
     }
 }
+print "Operative system found: $OS";
```

Comment:
I am printing what gets stored on the OS variable, (that holds the operative system)